app.go

```diff
@@ -60,11 +62,22 @@ func NewHeadscale(cfg Config) (*Headscale, error) {
 		return nil, err
 	}
 	pubKey := privKey.Public()
+
+	var dbString string
+	switch cfg.DBtype {
+	case "postgres":
+		dbString = fmt.Sprintf("host=%s port=%d dbname=%s user=%s password=%s sslmode=disable", cfg.DBhost,
+			cfg.DBport, cfg.DBname, cfg.DBuser, cfg.DBpass)
+	case "sqlite3":
+		dbString = cfg.DBpath
```

Comment:
Question - what do we want the default path to be for the sqlite db? Should we adopt the same logic as for the config file here and wrap this in an absPath() call? It's a little weird to have database files in a config directory though...